### PR TITLE
Refactor.tracelog

### PR DESCRIPTION
**Pull request by resoliwan**
1. make cls to optional (we can disable cls completely!)
2. refact mq trace log to draw mermaid flow digram.
3. apply tslint.

### CODE REVIEW

File: src/services/rpc-service.ts [hunk 1]
```diff
@@ -355,35 +354,35 @@ export default class RPCService {
...
       correlationId,
       expiration: Environments.ISLAND_RPC_WAIT_TIMEOUT_MS,
-      headers,
+      headers: this.getHeader(),
       replyTo: this.responseQueueName,
       timestamp: +(new Date())
     };
   }
 
+  private getHeader(): any {
```
Comment (heycalmdown) [inline src/services/rpc-service.ts]: How about to use `makeInvokeHeader()`?

File: src/services/rpc-service.ts [hunk 2]
```diff
@@ -412,24 +411,30 @@ export default class RPCService {
...
-      }
-    }
+    options.headers = this.getOptionsHeader(options);
     await this.channelPool.usingChannel(async channel => {
       return channel.sendToQueue(replyTo, RpcResponse.encode(value), options);
     });
     return value;
   }
 
+  private getOptionsHeader(options: amqp.Options.Publish) {
```
Comment (heycalmdown) [inline src/services/rpc-service.ts]: to use `makeReplyHeader()` as similar?

**Comment by kuyh** [comment]
LGTM

File: src/services/rpc-service.ts (hunk 1 above)
Comment (resoliwan) [reply src/services/rpc-service.ts]: Great!

File: src/services/rpc-service.ts (hunk 2 above)
Comment (resoliwan) [reply src/services/rpc-service.ts]: Gooooood!

**Pull request merged**